doc/onboarding.md

```diff
@@ -170,18 +170,24 @@ Landing a PR
 * `git rebase -i upstream/master`
 * squash into logical commits if necessary
 * `./configure && make -j8 test` (`-j8` builds node in parallel with 8 threads. adjust to the number of cores (or processor-level threads) your processor has (or slightly more) for best results.)
-* Amend the commit description
-  * commits should follow `subsystem[,subsystem]: small description\n\nbig description\n\n<metadata>`
-  * first line 50 columns, all others 72
-  * add metadata:
-    * `Fixes: <full-issue-url>`
-    * `Reviewed-By: human <email>`
-      * Easiest to use `git log` then do a search
-      * (`/Name` + `enter` (+ `n` as much as you need to) in vim)
-      * Only include collaborators who have commented `LGTM`
+* Amend the commit description.
+  * Commits should be of the form `subsystem[,subsystem]: small description\n\nbig description\n\n<metadata>`
+  * The first line should not exceed 50 characters.
```

Comment:
I like it, but that gets a little trickier and starts leaking into other docs:

* Commit message formatting requirements are in `CONTRIBUTING.md` and belong there, IMO. This doc should just link to that section (except for metadata format, since that's Collaborator-specific).

* The `CONTRIBUTING.md` should include the imperative mood requirement. I don't think it does right now. So it will need to be updated too.

* There may be some discussion around whether using the term `imperative mood` will be well-understood or is completely accurate. We may want to go with something more like "should start with a present-tense verb". Not sure but there seems to be a risk of bike-shedding.

For that reason, if it's OK with you, I'd prefer to make the whole adjusting-the-requirements-for-the-commit-message thing be a separate pull request. (Feel free to open it yourself if you wish!)